app/src/main/res/xml/customization_preferences.xml

```diff
@@ -44,4 +44,15 @@
             android:key="@string/pref_key_toolbar_bottom"
             android:title="@string/preference_bottom_toolbar" />
     </androidx.preference.PreferenceCategory>
+
+    <androidx.preference.PreferenceCategory
+        android:key="@string/pref_home_category"
```

Comment:
Usually when we are toggling the visibility, we start the preference off as `app:isPreferenceVisible="false"` so it won't flash out of existence on non nightly builds while creating

Reply:
Nice, this is a good to know! Thanks for the catch